docs/Concepts/Pruning.md

```diff
@@ -0,0 +1,13 @@
+description: Pruning
+<!--- END of page meta data -->
+
+# Pruning
+
+As the number of transactions increases, the world state grows, and storage can become a problem. Use pruning on any network to remove state tries that aren't necessary.
```

Comment:
Suggested change: 
Use pruning to reduce storage required for the chain.  As the number of transaction increases, the world state grows. Pruning removes state tries that aren't required.

Reply:
This is updated but with some additional changes from the tech review.